charts/scalardb/templates/scalardb/deployment.yaml

```diff
@@ -47,8 +46,34 @@ spec:
             - containerPort: 50051
             - containerPort: 8080
           env:
-          - name: JAVA_OPTS
-            value: "-Dlog4j.logLevel={{ .Values.scalardb.storageConfiguration.dbLogLevel }}"
```

Comment:
After this update, we need to specify the log level using `SCALAR_DB_LOG_LEVEL` instead of `JAVA_OPTS`.